fideslib/db/base_class.py

```diff
@@ -177,7 +177,7 @@ def get_by_key_or_id(
         """Retrieves db object by id, if provided, otherwise attempts by key"""
         db_obj: FidesBase | list[FidesBase] | None = None
         if data.get("id") is not None:
-            db_obj = cls.get(db=db, table_id=data["id"])
+            db_obj = cls.get(db=db, client_id=data["id"])
```

Comment:
I missed that you were using table_id on these base class methods too, not just ClientDetail.get so then `client_id` doesn't make so much sense here in this context.  So if you wanted the same name everywhere that's not id, maybe `object_id`?